core/src/v2/actix.rs

```diff
@@ -521,3 +525,120 @@ where
         new.update_definitions(name, map);
     }
 }
+
+macro_rules! json_with_status {
+    ($name:ident => $status:expr) => {
+        pub struct $name<T: Serialize + Apiv2Schema>(pub T);
```

Comment:
Let's get rid of the constraints in the type. I prefer having it in the impl instead. Also, let's have a default type.
```rust
pub struct $name<T = ()>(pub T);
```